src/cider/nrepl/middleware/test.clj

```diff
@@ -197,18 +197,30 @@
                               :message "Uncaught exception, not in assertion"})))
       (test/do-report {:type :end-test-var :var v}))))
 
+(defn- var-filter
+  [var include exclude]
+  (let [test-inclusion (if include
+                         #((apply some-fn include) (meta %))
+                        (constantly true))
+        test-exclusion (if exclude
+                         #((complement (apply some-fn exclude)) (meta %))
+                         (constantly true))]
+    (and (test-inclusion var)
+         (test-exclusion var))))
+
 (defn test-vars
   "Call `test-var` on each var, with the fixtures defined for namespace
   object `ns`. Filter vars with `selector` metadata marker when `selector` is present."
-  [ns selector vars]
+  [ns include exclude vars]
```

Comment:
You forgot to update the docstring. I guess it should mention that those should be seqs.